src/documentation/basic-opts.page

```diff
@@ -12,15 +12,15 @@ This document covers the basic operations provided by Uniqush. It won't cover th
 
 ## How to Communicate with Uniqush
 
-Uniqush runs as a stand along program, which means you don't need to write any code to link against uniqush. Uniqush listens a port when it is running. You can change the port in its configuration file. To send request to uniqush, you simply need to connect to that port, send an HTTP Post request with parameters inside the body.
+Uniqush runs as a stand alone service, which means you don't need to write any code to link against uniqush. Uniqush listens on a TCP port when it is running. You can change the port in its configuration file. To send request to uniqush, you simply need to connect to that port, send an HTTP POST request with the parameters inside the body.
 
 ## Adding Push Service Providers to a Service
 
-Without telling uniqush infomation about push service provider, a server can never send data through uniqush. Usually, to use a push service provider, uniqush need to know the type of that push service provider (e.g. [GCM], [APNS], Black Berry Push Service or Microsoft Push Notification Service for Windows Phone) and authentication information -- for example, authentication token for [GCM]; private/public keys for [APNS], etc.
+Without telling uniqush infomation about a push service provider, a server can never send data through uniqush. Usually, to use a push service provider, uniqush need to know the type of that push service provider (e.g. [GCM], [APNS], Black Berry Push Service or Microsoft Push Notification Service for Windows Phone) and authentication information -- for example, authentication token for [GCM]; private/public keys for [APNS], etc.
 
 You can add more than one push service provider information to a service, so that you can support multiple platforms.
 
-## Subscribe a Service
+## Subscribing to a Service
 
 When a subscriber subscribes a service, uniqush need to know at least three things: the subscriber's name; the service name; and the delivery point information. The delivery point information including the identity of the device, its OS type and other push service provider specific information. For example, for android delivery points, they need to provide a registration id to uniqush, which will be used to send data through [GCM] later.
```

Comment:
more nits: "subscribes *to* a service", say subscriber's id instead of subscriber's name